gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -91,6 +91,19 @@ typedef enum
   ACCL_NPU_SR       = 0x4100,     /**< specify device as any SR npu, if possible */
 } accl_hw;
 
+/**
+ * @brief Internal tensor layout format for other/tensor
+ */
```

Comment:
We need more background info on this. Why do we need this / When do we need this / ...

Reply:
Added.